### ISSUE TITLE
MGDeathRun

### ISSUE THREAD

**Issue by anemul** [MysticCity/MinigamesAPI#149]
I'm not sure if it is suppose to do this after clicking and then trying to move or not . I am in the DeathRun lobby , I'm not sure why we would not be able to move around after clicking the join sign . Here is a movie to show you what I am talking about :

**Comment by anemul** [comment]
[DeathRun.zip](https://github.com/MysticCity/MinigamesAPI/files/442838/DeathRun.zip)

**Comment by mepeisen** [comment]
This is "working as designed". However you are running into a strange situation. The minigames lib does not like to let you run away. So whenever running to the lobby bounds it kicks you backwards.

It seeems that your Lobby bounds are too small. So while being kicked you ran over the other sides bounds and will be kicked back again.

Workaround is easy. As you already have some kind of wall/ room simply set the lobby bounds one block outside this room. So that the players can walk around but only inside this room.

Did it help?
I will leave this bug open because in version 2 this may be changed a bit. Administrators are more flexible in gaming rules and can for example apply the rule "do not move inside lobby" if they like to.

**Comment by anemul** [comment]
Yes it worked , but in the commands list it says those bounds are optional , maybe have to say mandatory . Thanks for the quick help .
